src/CameraControls.ts

```diff
@@ -835,7 +835,7 @@ export class CameraControls extends EventDispatcher {
 		bb.min.y -= paddingBottom;
 		bb.max.x += paddingRight;
 		bb.max.y += paddingTop;
-		
+
```

Comment:
The linter complained about this. Is the linting running in CI?

Reply:
uh oh thanks, I should set up the CI...